packages/opa-react/tsconfig.json

```diff
@@ -1,5 +1,6 @@
 {
   "compilerOptions": {
+    "lib": ["ESNext.Object", "DOM"],
```

Comment:
Needed to add something to be able to use `Object.groupBy`. Once I had added `ESNext.Object`, I list `RequestInit`, and had to add `DOM`.